### PR TITLE
Create troubleshoot-report-server.md

### PR DESCRIPTION
**Pull request by todorarabadzhiev**

### CODE REVIEW

File: knowledge-base/troubleshoot-report-server.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<table>
+	<tr>
+		<td>Product</td>
+		<td>Progress® Telerik® Report Server</td>
+	</tr>
+</table>
+
+
+## Description
+Sometimes it is possible to experience problems with the Report Server the reasons for which are not apparent. For example, restricted permission for accessing the File Storage, Internal Server Errors due to unaccessible Storage or missing/corrupted report resources (e.g. invalid image), etc.
```
Comment (atkeranov) [inline knowledge-base/troubleshoot-report-server.md]: unaccessible -> inaccessible / not accessible

File: knowledge-base/troubleshoot-report-server.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Solution
+We recommend using the following basic approaches for troubleshooting :
+  
+1. Check whether there are any errors in the browser console.  
+
+2. Use [Fiddler ](https://www.telerik.com/download/fiddler) to see the exact requests from the Report Manager Viewer to the server, and the corresponding responses - they may indicate what has failed.
```
Comment (atkeranov) [inline knowledge-base/troubleshoot-report-server.md]: What is a Report Manager Viewer ?

File: knowledge-base/troubleshoot-report-server.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+## Solution
+We recommend using the following basic approaches for troubleshooting :
+  
+1. Check whether there are any errors in the browser console.  
+
+2. Use [Fiddler ](https://www.telerik.com/download/fiddler) to see the exact requests from the Report Manager Viewer to the server, and the corresponding responses - they may indicate what has failed.  
+
+3. Attach a [Trace Listener](https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/how-to-create-and-initialize-trace-listeners) to the Report Server. The generated log should contain the entire Stack Trace of the error causing the failure. This can be done in the _Web.config_ file of the Report Server application. By default, it is in the (_Telerik Report Server installation folder_)\\Telerik.ReportServer.Web (e.g. _C:\\Program Files (x86)\\Progress\\Telerik Report Server\\Telerik.ReportServer.Web_). The code for enabling the _Trace Listener_ may look like :
```
Comment (atkeranov) [inline knowledge-base/troubleshoot-report-server.md]: What about a trace listener in the scheduling service?

File: knowledge-base/troubleshoot-report-server.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            <add name="myListener" type="System.Diagnostics.TextWriterTraceListener" initializeData="c:\temp\ReportServer.LOG" />            
+            <remove name="Default" />
+          </listeners>
+        </trace>
+    </system.diagnostics>
+	...
+</configuration>
+```
+  
+If the problem persists, you may open a support ticket and send us the [SAZ file](https://docs.telerik.com/fiddler/Save-And-Load-Traffic/Tasks/CreateSAZ) recorded by _Fiddler_ and the log file generated by the _Trace Listener_ while experiencing the issue.
```
Comment (silviyaboteva) [inline knowledge-base/troubleshoot-report-server.md]: Maybe it is more suitable to say: "After you generate the log files from the above steps, archive them and attach them to a support ticket. Include the steps which have to be followed in order to reproduce the issue."

File: knowledge-base/troubleshoot-report-server.md (hunk 1 above)
Comment (todorarabadzhiev) [reply knowledge-base/troubleshoot-report-server.md]: corrected

File: knowledge-base/troubleshoot-report-server.md (hunk 2 above)
Comment (todorarabadzhiev) [reply knowledge-base/troubleshoot-report-server.md]: changed to "...requests from the Viewer of the Report Server Manager to the Report Server REST Service..."

File: knowledge-base/troubleshoot-report-server.md (hunk 4 above)
Comment (todorarabadzhiev) [reply knowledge-base/troubleshoot-report-server.md]: The paragraph is modified partly

File: knowledge-base/troubleshoot-report-server.md (hunk 3 above)
Comment (todorarabadzhiev) [reply knowledge-base/troubleshoot-report-server.md]: To be included

Comment (todorarabadzhiev) [reply knowledge-base/troubleshoot-report-server.md]: Done

**Pull request merged**